packages/proto-loader/bin/proto-loader-gen-types.ts

```diff
@@ -844,7 +845,6 @@ async function runScript() {
     .default('bytes', 'Buffer')
     .default('inputTemplate', `${templateStr}`)
     .default('outputTemplate', `${templateStr}__Output`)
-    .default('outputBranded', false)
```

Comment:
If you add more than two defaults, typescript compiler will complain about excessive type instantiation.

<img width="711" alt="image" src="https://user-images.githubusercontent.com/15868333/204966901-bc0a2443-9fa0-4f5b-9ab9-67cb28576b5a.png">

I think this is because of the depth limitation of recursive type inference. Seems like we already reached the maximum number of inferable parameters.

To mitigate this I removed defaults for `inputBranded` and `outputBranded` in this commit, making them optional.

Although they would still be treated as falsy by default(undefined), help description will not tell that they will be treated as falsy by default. This might confuse users.